python/lsst/ts/scheduler/model.py

```diff
@@ -0,0 +1,46 @@
+import time
+from SALPY_scheduler import *
+
+DEFAULT_STATE = "OFFLINE"
+
+class Model():
+	"""The model contains all business logic related to the scheduler.
```

Comment:
This file uses tab as indentation. Need to fix it to comply with the other files.

Reply:
hmmm I am wondering how this is happening, I thought it was my remote editor that did that but I fixed it... regardless, done